test/memcache-helper.js

```diff
@@ -10,8 +10,7 @@ P.promisifyAll(Memcached.prototype)
 
 var config = {
   memcache: {
-    host: '127.0.0.1',
-    port: '11211'
+    address: '127.0.0.1:11211',
```

Comment:
what does this changed do?

Reply:
'address' is being used here but never defined :
https://github.com/mozilla/fxa-customs-server/blob/be3b5794da2df8d8d7d042a269ef3ca47e248c1f/test/memcache-helper.js#L41